domain-models-runtime/src/main/java/org/folio/rest/tools/parser/JsonPathParser.java

```diff
@@ -274,7 +274,7 @@ private Object getValueAt(String path, Object o, Pairs parent, boolean returnRoo
           parent.rootNode = o;
         }
       }
-      if(returnRoot){
+      if(returnRoot && parent != null){
```

Comment:
```suggestion
      if (returnRoot && parent != null) {
```

Reply:
@adamdickmeiss , committed suggested changes. thanks.

Reply:
Can you push the commit?